modules/js/CardTrait.js

```diff
@@ -177,6 +177,9 @@ define(['dojo', 'dojo/_base/declare'], (dojo, declare) => {
         if (n.args.src == 'deck') {
           // Make sure it will pass in front on discard
           dojo.style('bang-card-' + card.uid, 'zIndex', dojo.query('#discard .bang-card').length);
+          dojo.query('#bang-card-' + card.uid + ' .card-background').forEach((cardBackground) => {
```

Comment:
This is a js fix, but it shouldn't be needed. Were you able to reproduce the issue ?
It might be related to safari that needs a prefix for backface-visilibity.

Reply:
I don't have Safari and never had Safari in my life. And 95% of time I see those ugly Schofields. In Chrome. Thaat's what majority of users see